src/containers/Auth/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,156 @@
+
+
+/**
+ * @class ResetPassword
+ * @description Component to send reset password link to users.
+ */
+export class ResetPassword extends Component {
+  static propTypes = {
+    toastManager: PropTypes.object.isRequired,
+    response: PropTypes.string,
+    error: PropTypes.bool,
+    sendResetLink: PropTypes.func,
+    isLoading: PropTypes.bool,
+    success: PropTypes.bool,
+  }
+
+  /**
+   * @constructor
+   * @description holds the state;
+   */
+  constructor() {
+    super();
+    this.state = {
+      error: {},
+      email: '',
+    };
+  }
+
+  /**
+   * @param {*} e - event that is acted upon
+   * @returns {object} - The changed state
+   */
+  handleInputChange = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  }
+
+  /**
+   * @description - decides if component should throw error or update
+   * @param {*} nextProps
+   * @param {*} nextState
+   * @returns {bool} - if the component should be uppdated or not
+   */
+  shouldComponentUpdate(nextProps) {
+    if (this.props.error !== nextProps.error && nextProps.error === true) {
+      this.props.toastManager.add(`${nextProps.response}`, {
+        appearance: 'error',
+        autoDismiss: true,
+      });
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   *
+   * @param {*} e The event that is to be submitted
+   * @returns {func} - calls the send email function
+   */
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { sendResetLink } = this.props;
+    sendResetLink(this.state.email);
+  }
+
+  /**
+   * @function
+   * @returns {JSX} The reset password form JSX
+   * or the reset password email successfully sent JSX
+   */
+  render() {
+    return (
+        <span>
+        <FormWrapper imageUrl={image} flexWrapper='reset-flex-wrapper'
+        imgId='sideImage' displayImage={false} >
+            {
+          !this.props.success
+            ? (<FormContainer
+            header="Forget Password"
+            containerId='resetContainer'
+            loginId='resetFormWrapper'
+            onSubmit={this.handleSubmit}>
+              <div className="input-group">
+                  <label className='reset-password-label'
+                  id='resetLabel'>Email Address</label>
+                  <Input
+                      type="email"
+                      placeholder="example@example.com"
+                      id="email"
+                      classes='reset-password-tbx'
+                      required={true}
+                      onChange={this.handleInputChange}
+                    />
+                </div>
+                <div className='text-center' id='clipLoader'>
+                {!this.props.isLoading
+                  && <Button
+                  type={'submit'} buttonId='submitButton'
+                  classes='form-button' >
+                  Send Reset Password Link</Button>}
+
+                  <ClipLoader
+                    css={'override'}
```

Comment:
@idmega2000 is there an `override` class? if not, you can remove the css field.
`size unit` can just be `px` without curly brace